_posts/2017-08-10-rubyonrails.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Learn Rails and build web apps"
```

Comment:
I was talking about "build" as well, but I think either of WebApps, Webapps and Web Apps should be fine